jwst/wfs_combine/wfs_combine.py

```diff
@@ -252,31 +252,19 @@ def get_wcs_offsets(self):
         off_y: integer
             difference (#2 -#1) in pointing in pixels in the y-direction
         """
-
-        #crval1 = self.input_1.meta.wcs.crval
-        #crval2 = self.input_2.meta.wcs.crval
-        #crpix2_in_1 = self.input_1.meta.wcs.invert(*crval2)
-        #crpix = self.input_1.meta.wcs.crpix
-
-        crval1_model = self.input_1.meta.wcs.forward_transform['crval']
-        crval1 = crval1_model.lon.value, crval1_model.lat.value
-        crval2_model = self.input_2.meta.wcs.forward_transform['crval']
-        crval2 = crval2_model.lon.value, crval2_model.lat.value
-        # exclude the distortion as the inverse does not give sensible numbers yet
-        # but may be include it after this is fixed?
-        fits_wcs_transform = self.input_1.meta.wcs.forward_transform['crpix1':]
-        #crpix2_in_1 = self.input_1.meta.wcs.backward_transform(*crval2)
-        crpix2_in_1 = fits_wcs_transform.inverse(*crval2)
-        crpix = np.array((self.input_1.meta.wcs.forward_transform['crpix1'].offset.value,
-                self.input_1.meta.wcs.forward_transform['crpix2'].offset.value), dtype=np.float)
-        off_x, off_y = crpix2_in_1 + crpix
-
-        log.info('CRVAL of Image 1 in WCS of #1 has pixel coords:%s %s',
-                 crval1[0], crval1[1])
-        log.info('CRVAL of Image 2 in WCS of #2 has pixel coords:%s %s',
-                 crval2[0], crval2[1])
-        log.info('Image 2 has these nominal offsets relative to image 1\
-                 (x,y): %s %s', off_x, off_y)
+        wcs1 = self.input_1.meta.wcs
+        wcs2 = self.input_2.meta.wcs
+        tr1 = wcs1.get_transform('detector', 'world')
```

Comment:
This is equivalent to calling just calling `wcs` as a function, e.g. `wcs(xcen, ycen)`.

Reply:
That's what I thought.  I suggested explicitly specifying frames because when I tried just calling wcs I saw something that implied it was mapping to 'v2v3' instead of to 'world' (it was a warning message when I called wcs with too few arguments).  If I understand correctly, we really need the mapping from 'detector' to 'world', though perhaps it would work to map between 'v2v3' and 'world' (but definitely not from 'detector' to 'v2v3').

Reply:
That makes sense. If `world` is not in the WCS, then it will fail. Perhaps this is what needs to happen.

Reply:
FYI: This is what happens with the outdated regression test data. 👍

Reply:
There was no pointing information at the time. I think this kind of data will fail in the assign_wcs step if it doesn't have the pointing info. The problem with the test is it had a stale WCS from years ago.